Tests/test_pairwise_aligner.py

```diff
@@ -18621,6 +18623,21 @@ def test_pickle_aligner_substitution_matrix(self):
             pickled_aligner.extend_right_deletion_score,
         )
         self.assertEqual(aligner.mode, pickled_aligner.mode)
+        self.assertEqual(aligner.epsilon, pickled_aligner.epsilon)
+        self.assertEqual(aligner.algorithm, pickled_aligner.algorithm)
+
+    def test_pickle_aligner_alignment_symmetry(self):
```

Comment:
Can you change "symmetry" to something like "consistent"? or just leave it out altogether. The word "symmetry" does not quite apply here.